src/styles/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $calendarPadding;
+  gap: 1px;
+  width: calendarWidth();
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $dayWidth;
+    height: $dayHeight;
+    background-color: #eee;
+    border: 1px solid black;
+    transition-duration: 0.5s;
+
+    @for $date from 1 through 31 {
+      &:nth-child(#{$date})::before {
+        content: "#{$date}";
+        font-family: Arial, sans-serif;
+        font-size: 30px;
```

Comment:
it may be better to assign these styles for all page in general rather than in a loop